drainer/sync/kafka.go

```diff
@@ -22,10 +22,11 @@ import (
 	"github.com/Shopify/sarama"
 	"github.com/pingcap/errors"
 	"github.com/pingcap/log"
-	"github.com/pingcap/tidb-binlog/drainer/translator"
-	"github.com/pingcap/tidb-binlog/pkg/util"
 	obinlog "github.com/pingcap/tidb-tools/tidb-binlog/proto/go-binlog"
 	"go.uber.org/zap"
+
+	"github.com/pingcap/tidb-binlog/drainer/translator"
```

Comment:
Why we need this order change? Could you please revert it?

Reply:
It's auto-adjusted by fmt. Most directory import order in tidb-binlog is like this.